logic.py

```diff
@@ -74,15 +74,17 @@ def set_apc(**kwargs):
         except (ObjectDoesNotExist, IndexError):
             pass
 
+
 def notify_billing_staffers(**kwargs):
     request = kwargs.get('request', None)
     article = kwargs.get('article', None)
+    type_of_notification = kwargs.get('type_of_notification', 'ready')
```

Comment:
As far as I can see, the only way we get the 'ready' notification to fire is with the default on this `.get()`
It seems very implicit to have it work this way.

That said, I can't think of a better way to link the notification type to the event registered for this function (`ON_ARTICLE_ACCEPTED` I believe?).